tests/tests.rs

```diff
@@ -58,3 +58,7 @@ mod test_shell;
 fn rustc_host() -> String {
     cargo::ops::rustc_version("rustc").unwrap().1
 }
+
+fn is_nightly() -> bool {
+    cargo::ops::rustc_version("rustc").unwrap().0.contains("-nightly")
```

Comment:
Would not a development build (`-dev`) would satisfy the intent of the `is_nightly` check also (*viz.* using unstable features)?